lib/util/tooltip.coffee

```diff
@@ -31,9 +31,13 @@ class Tooltip
     @showOnHover()
     this
 
+  hide_: ->
```

Comment:
I think we could animate `padding` and `max-height`, but then I don't think that's worth it for such a short animation...